src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalTests.cs

```diff
@@ -3742,5 +3742,383 @@ public void TestEqualsConflictMarker2()
             Assert.Equal(36, trivia4.Span.Start);
             Assert.Equal(24, trivia4.Span.Length);
         }
+
+        [Fact]
+        public void TestEqualsConflictMarker3()
+        {
+            // second ======= is part of disabled text
+            var token = Lex("======= Trailing\r\ndisabled text 2\r\n======= \r\nmore disabled text\r\n>>>>>>> Actually the end").First();
```

Comment:
I know the rest of these tests are using the single-line syntax, but can we use raw strings here for readability? I find it very hard to understand what this is testing.

Reply:
a main problem with the compiler test codebase is that its git rules transforms the content to have unix vs windows newlines when going between systems.  So it's hard to actually test/validate things like this with raw strings since the test input literally changes depending on the system you run it on.  You'd need to have the test validation do the same to make up for this.

My preference would be that files are data and for many reasons (consistency, determinism, etc.) the contents are the same no matter what system you are on.